pl_engine/pl_server/src/main/java/com/cubrid/jsp/value/DateTimeParser.java

```diff
@@ -47,26 +47,29 @@
 
 public class DateTimeParser {
 
-    private static final ZoneOffset TIMEZONE_0 = ZoneOffset.of("Z");
-    // TODO: update the following value along with the server
-    private static final ZoneOffset TIMEZONE_SESSION = ZoneOffset.of("+09:00");
-
     // zoneless part of min timestamp: 1970-01-01 00:00:01
-    private static final LocalDateTime minTimestampLocal = LocalDateTime.of(1970, 1, 1, 0, 0, 1);
+    public static final LocalDateTime minTimestampLocal = LocalDateTime.of(1970, 1, 1, 0, 0, 1);
     // zoneless part of max timestamp local part: 2038-01-19 03:14:07
-    private static final LocalDateTime maxTimestampLocal = LocalDateTime.of(2038, 1, 19, 3, 14, 7);
+    public static final LocalDateTime maxTimestampLocal = LocalDateTime.of(2038, 1, 19, 3, 14, 7);
+
     // min datetime: 0001-01-01 00:00:00.000
-    private static final LocalDateTime minDatetime = LocalDateTime.of(1, 1, 1, 0, 0, 0, 0);
+    public static final LocalDateTime minDatetimeLocal = LocalDateTime.of(1, 1, 1, 0, 0, 0, 0);
     // max datetime: 9999-12-31 23:59:59.999
-    private static final LocalDateTime maxDatetime =
+    public static final LocalDateTime maxDatetimeLocal =
             LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999);
 
+    private static final ZoneOffset TIMEZONE_0 = ZoneOffset.of("Z");
+    // TODO: update the following value along with the server
+    private static final ZoneOffset TIMEZONE_SESSION = ZoneOffset.of("+09:00");
```

Comment:
It will be reflected default timezone from server in the future (maybe phase 1 or 2)

Reply:
Yes. This is a temporary hard-coding